Platform/ElkhartlakeBoardPkg/Library/Stage2BoardInitLib/Stage2BoardInitLib.c

```diff
@@ -1600,7 +1603,11 @@ UpdateFspConfig (
 
   if (mPchSciSupported) {
     Fspscfg->IsFusaSupported = 0x1;
-    Fspscfg->IehMode = 0x1;
+    if (MemCfgData != NULL) {
```

Comment:
Should null check be more an ASSERT? or can memcfgdata be actually null?

in case if statement is used please consider add an else condition that sets
Fspscfg->IehMode = 0x1;
to keep same behavior as before.